server/chat.ts

```diff
@@ -1107,6 +1107,29 @@ export class CommandContext extends MessageContext {
 
 		return message;
 	}
+	canPMHTML(targetUser: User | null) {
+		if (!targetUser || !targetUser.connected) {
+			this.errorReply(`User ${this.targetUsername} is not currently online.`);
```

Comment:
(editors note: this is written like this so it's theoretically easier to refactor all the perm checks to use `ErrorMessage`s)